spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -16,13 +16,13 @@
       ems.update_authentication(:default => {:userid => username, :password => password})
     end
   end
-  let(:collector) { described_class.new(ems) }
+  let(:collector) { ManageIQ::Providers::Vmware::InfraManager::Inventory::Collector.new(ems) }
```

Comment:
Because the targeted refresh tests call out to the collector to process targeted watch updates we still need a collector instance